jobs/build/ose/scripts/merge-and-build.sh

```diff
@@ -339,12 +371,46 @@ ose_images.sh --user ocp-build compare_nodocker --branch rhaos-${OSE_VERSION}-rh
 
 echo
 echo "=========="
+echo "Run OIT rebase"
+echo "=========="
+
+cat >"${OIT_WORKING}/sources.yml" <<EOL
+ose: ${OSE_DIR}
+jenkins: ${OPENSHIFT_JENKINS_DIR}
+openshift-ansible: ${OPENSHIFT_ANSIBLE_DIR}
+EOL
+
+${OIT_PATH} --working-dir ${OIT_WORKING} --group openshift-${OSE_VERSION} \\
+--include aos3-installation-docker \\
```

Comment:
Needs to be the --optional-include (does not exist yet) we discussed since several releases don't include the jenkins images.

Reply:
Yes... right. Will add that into OIT shortly.